CommunicationLayer/CommDeviceControl/UdpMessageForwarder.h

```diff
@@ -25,9 +25,12 @@
 
 #pragma once
 
+#include <SimpleAmqpClient/SimpleAmqpClient.h>
 #include <QUdpSocket>
 #include "CommunicationLayer/CommDeviceControl/I_MessageForwarder.h"
 
+using namespace AmqpClient;
```

Comment:
The `using` directive is generally considered harmful because it causes keywords to pop-up which are hard to tell where they're from.